dashboard/app/helpers/levels_helper.rb

```diff
@@ -775,6 +775,7 @@ def level_title
         script
       end
     elsif @level.try(:is_project_level) && data_t("game.name", @game.name)
+      # This is updated in project.js with the name of the project
```

Comment:
Could you paste a link to the code you're referencing?

Reply:
https://github.com/code-dot-org/code-dot-org/blob/staging/apps/src/code-studio/initApp/project.js#L661-L665. The reason I didn't put a link in the comments is because the line number is likely to change, was there something more specific you wanted in the comment?